milli/src/update/index_documents/mod.rs

```diff
@@ -1489,6 +1489,7 @@ mod tests {
         assert_eq!(count, 4);
     }
 
+    #[cfg(feature = "default")]
```

Comment:
Why have you added this `cfg` ?

Reply:
ah, that shouldn't have been part of this PR. But since https://github.com/meilisearch/milli/pull/632 , milli can't tokenise all languages unless the default features are activated. This test needs one of those optional tokenisers to succeed but it wasn't feature-gated, so it failed when run with `cargo test --no-default-features`